backend/src/services/user.go

```diff
@@ -19,3 +20,7 @@ type UserService struct {
 func (u *UserService) GetAllUsers() ([]models.User, error) {
 	return transactions.GetAllUsers(u.DB)
 }
+
+func (u *UserService) GetUser(id string) (*models.User, error) {
+	return transactions.GetUser(u.DB, id)
```

Comment:
what if id is "foo" what if id is "-1", let's do some validation here by converting id to a uint. (hint postgre ids start at 1)

Reply:
This is sanatized in the controller. On Wednesday you guys told us to keep it there, but it can certainly be refactored into the service.

Reply:
oh you're right my bad, @DOOduneye it's fine to leave that in the controller then, right?

Reply:
done